rasa/engine/recipes/default_recipe.py

```diff
@@ -374,9 +374,11 @@ def _extra_config_from_cli(
             old_number_epochs = component_config.get(
                 EPOCHS, component.get_default_config()[EPOCHS]
             )
-            epoch_fraction = float(cli_parameters["finetuning_epoch_fraction"])
-
-            config_from_cli[EPOCHS] = math.ceil(old_number_epochs * epoch_fraction)
+            epoch_fraction = cli_parameters["finetuning_epoch_fraction"]
```

Comment:
how about we 
 ```suggestion
            epoch_fraction = cli_parameters.get("finetuning_epoch_fraction", 1.0)
```
and drop the "if" to make it more explicit that the default is 1.0 (in case we need to find that again one day ^^). Ideally, we should grab the default from some `constants.py`, but seems like there is none for that? 👀

Reply:
+1 to adding constants for `1.0` and `finetuning_epoch_fraction`